evap/evaluation/models.py

```diff
@@ -362,12 +370,12 @@ def textanswer_set(self):
     @property
     def open_textanswer_set(self):
         """Pseudo relationship to all text answers for this course"""
-        return TextAnswer.objects.filter(contribution__in=self.contributions.all(), checked=False)
+        return TextAnswer.objects.filter(contribution__in=self.contributions.all(), state='')
```

Comment:
we could also use any other character like a ```-```, but i think an empty character defines very good that no decision on the publishing state has been made yet.